troposphere/static/js/components/admin/Emulate.jsx

```diff
@@ -0,0 +1,41 @@
+import $ from "jquery";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Glyphicon from 'components/common/Glyphicon';
+import Tooltip from "components/common/ui/Tooltip";
```

Comment:
I think what I hit was that the tooltip being renders with repeated elements (in a table) seemed to behave oddly. 

I'll get a branch using it up so you can see and I'll record a screen-cap too.